### ISSUE TITLE
Graphical view size of connectors version 1.22

### ISSUE THREAD

**Issue by jobehrendt** [OpenModelica/OpenModelica#11380]
### Description
This bug is relevant for the version 1.22 which was delivered as stable version for some weeks.
In the graphical view the connectors of submodels were display very big so that no further editing is possible.  I have reproduced this error in the Linux (Debian bookworm) and Windows 10 version as well. 

### Steps to reproduce
I am providing you with an example code (as zip) and two screenshots to 
[Tracer_Graph.zip](https://github.com/OpenModelica/OpenModelica/files/12914136/Tracer_Graph.zip)
![Versin2 21](https://github.com/OpenModelica/OpenModelica/assets/13729055/91d0d6d8-1af9-41b5-9dbe-163ea8c2fd3f)
![Version1 22](https://github.com/OpenModelica/OpenModelica/assets/13729055/a0799ed4-59cb-4886-9629-4bd731647ca6)

I hope you can fix this issue. Thanks for providing us with OpenModelica.
Best regards 
Joachim Behrendt

Example script `reproduceBug.mos`:
```modelica
getVersion(); getErrorString();
loadModel(Modelica,{"4.0"}); getErrorString();
loadFile("path/to/MyLibrary");getErrorString();
setCompilerFlags("-d=optdaedump");getErrorString();
simulate(MyLibrary.MyModel); getErrorString();
```
You need to run the script from the command line interface. On Linux, you can use your favourite shell, cd to the directory where the .mos file is located and run
```
$ omc reproduceBug.mos
```
On Windows, start the command line interface (right-click on the start menu, then run the CMD program), cd to the directory where the .mos file is located and type
```
C:\MyDirectory>"C:\Program Files\OpenModelica1.18.0\bin\omc.exe" reproduceBug.mos
```

You may need to adapt the path based on the actual installed version of OpenModelica.

We really 💖 [minimal working examples](https://en.wikipedia.org/wiki/Minimal_working_example). If you can provide a MWE that shows the problem, instead of your original model, reproducing the bug is more straightforward and we can respond faster.

### Additional files
You can add additional files via drag-and-drop, e.g. for your minimal working example. Please note that GitHub does not accept .mo and .mos attachments, so you can either zip the example files and attach the .zip file, or add .txt to the extension of a simple .mo or .mos file.

You can also use `File->Save Total` in OMEdit to save a model and all its dependencies to a single file, so you don't need to provide all the libraries you are using. If your model is sensitive or confidential, you can use the obfuscation option, that removes all documentation and scrambles the variable names, preventing reverse-engineering.

### Version and OS
 - OpenModelica Version [run `omc --version` or check `Help->About OMEdit` from OMEdit]
 - Versions of used Modelica libraries if applicable
 - OS: [e.g. Windows 10, 64 bit]

**Comment by casella** [comment]
@adeas31 maybe this could still end up in 1.22.0?

**Comment by adeas31** [comment]
Yes. This is happening because the connectors have a line thickness of 3 instead of default 0.25. I don't know yet how to fix it.

**Comment by casella** [comment]
@adeas31 should we make a beta.5, or can we trust that it works out of the box?

Maybe you could just compile the maintenance branch yourself and double-check it works.

**Comment by adeas31** [comment]
We can do a beta5. I already tested the maintenance branch.

**Comment by adeas31** [comment]
We can do a beta5. I already tested the maintenance branch.

**Comment by jobehrendt** [comment]
with the beta 6 it works out better, Thanks for considering my comment.

**Comment by casella** [comment]
Thank you for pointing this out! 👍
